packages/better-auth/src/plugins/organization/client.ts

```diff
@@ -42,35 +41,7 @@ interface OrganizationClientOptions {
 				enabled: boolean;
 		  }
 		| undefined;
-	schema?:
-		| {
-				organization?: {
-					additionalFields?: {
-						[key: string]: DBFieldAttribute;
-					};
-				};
-				member?: {
-					additionalFields?: {
-						[key: string]: DBFieldAttribute;
-					};
-				};
-				invitation?: {
-					additionalFields?: {
-						[key: string]: DBFieldAttribute;
-					};
-				};
-				team?: {
-					additionalFields?: {
-						[key: string]: DBFieldAttribute;
-					};
-				};
-				organizationRole?: {
-					additionalFields?: {
-						[key: string]: DBFieldAttribute;
-					};
-				};
-		  }
-		| undefined;
+	schema?: OrganizationOptions["schema"];
```

Comment:
As an alternative, if we don't want to explicitly allow the use of all other server supported options, we can extend the existing client options to allow any other field, e.g:

```diff
organization?: {
					additionalFields?: {
						[key: string]: DBFieldAttribute;
					};
+                    [key: string]: any,
				};
```